xarray/backends/api.py

```diff
@@ -1125,7 +1125,14 @@ def dump_to_store(
 
 
 def save_mfdataset(
-    datasets, paths, mode="w", format=None, groups=None, engine=None, compute=True
+    datasets,
+    paths,
+    mode="w",
+    format=None,
+    groups=None,
+    engine=None,
+    encodings=None,
```

Comment:
Isn't this technically a breaking change since you added it before `compute`? It seems to make sense though if you look at other methods. The annoying part of using positional arguments...

Reply:
We could disallow more than two positional arguments here. We did this in `open_dataset` for the backends refactor (without a deprecation period, directly raising an error).

https://github.com/pydata/xarray/blob/3666a6fb7226601002f408d56e81837ab69b8d4e/xarray/backends/api.py#L466-L469